src/Honeybee_Generator_batteries_Simple.py

```diff
@@ -0,0 +1,205 @@
+# By Anton Szilasi
```

Comment:
This component is going to be for future versions of Honeybee generation (At the moment is is only for grid connected electricity not for storage systems) so don't merge it into your master yet